libraries/lambda_handlers/ts_v2_catalog_handler.py

```diff
@@ -795,7 +796,7 @@ def _build_catalog_node(self, catalog, language, resource, project, modified):
             'tw_cat': '{}/{}/{}/{}/tw_cat.json?date_modified={}'.format(
                 self.cdn_url,
                 TsV2CatalogHandler.cdn_root_path,
-                pid, lid, r_modified)
+                pid, lid, r_modified)})
```

Comment:
This was a syntax error that I don't know when it was introduced but it is fixed now.